Jenkinsfile

```diff
@@ -21,10 +21,10 @@ properties([buildDiscarder(logRotator(artifactNumToKeepStr: '5', numToKeepStr: e
 
 def buildOs = 'linux'
 def buildJdk = '8'
-def buildMvn = '3.9.9'
+def buildMvn = '3.x.x'
```

Comment:
There is no label `3.9.9` in method `mvnFromVersion`

https://github.com/apache/maven-jenkins-env/blob/master/vars/jenkinsEnv.groovy#L62-L83